### PR TITLE
Milestone :  Created work and about section

### PR DESCRIPTION
**Pull request by Roseokpe**
#Mileston: Portfolio mobile version 2

In this milestone I created the : 
- Work section
- Contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -107,6 +107,99 @@ <h4>Skills</h4>
...
+          I’d love to help with it
+        </p>
+      </div>
+      <form>
+        <input type="text" name="text" placeholder="Text Input">
+        <input type="email" name="email" id="email" placeholder="test@gmail.com">
+        <textarea name="text" id="text" cols="30" rows="5">Enter text here</textarea>
+        <button type="submit">Send</button>
+      </form>
     </section>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please remove the contact form section from this milestone is not part of the requirement for this milestone.

File: style.css [hunk 2]
```diff
@@ -158,3 +158,175 @@ body {
...
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  border-right: 1px solid #555;
+
+  /* padding: 8px 0; */
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Roseokpe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️

- [ ] Kindly follow the microverse [README TEMPLATE](https://github.com/microverseinc/readme-template) to update your different readme headers to make your project more readable and professional.

- [ ] You nicely follow the project design and required features but your app is not responsive to mobile screen size.
### **Take a look**
mobile


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -151,10 +158,190 @@ body {
...
-  position: static;
   width: 18px;
   height: 21px;
   left: 0;
   top: 9.35px;
   margin: 0 12px;
 }
+
+.work-container {
+  display: flex;
```
Comment (gtekle) [inline style.css]: - [x] You should use CSS Grid layout for the works container as in the project requirement
css-grid-flexbox
.

File: index.html [hunk 4]
```diff
@@ -108,5 +108,83 @@ <h4>Skills</h4>
         </div>
       </div>
     </section>
+    <!----      Work section                    ---- -->
+      <section class="work-container">
+        <div class="work-tittle">
+          <h1>MY RECENT <br><br> WORK</h1>
```
Comment (gtekle) [inline index.html]: - [x] You shouldn't use more than one `<h1>` tag in an HTML document for semantic reasons. And you have already used on in the headline section.

File: index.html [hunk 5]
```diff
@@ -108,5 +108,83 @@ <h4>Skills</h4>
...
+                <button class="bttn" type="button">See Project</button>
+            </div>
+          </div>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
+            <div class="support-text">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.<br>
+                has been the industry's standard dummy text ever since the 1500s,<br>
+                when an unknown printer took a standard dummy text.
```
Comment (gtekle) [inline index.html:175]: - [ Optional ] You should use a `font-family` of Inter and `font-size` of 20px for the project description text.

File: index.html [hunk 6]
```diff
@@ -108,5 +108,83 @@ <h4>Skills</h4>
...
+                  <li> html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby </li>
+                </ul>
+                <button class="bttn" type="button">See Project</button>
+            </div>
+          </div>
+          <div class="projects-seection">
+            <a href=""><img src="images/Img-Placeholder.png" alt=""></a>
+            <h2 class="post-tittle">Multi-Post Stories</h2>
```
Comment (gtekle) [inline index.html:171]: - [ Optional ] You should use a `font-family` of `Roboto` and `font-size` of 32px for the project title text.

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @Roseokpe :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
this-close


 ## Highlights
:heavy_check_mark: Nice UI
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Clean code

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] You should add a professional README, as the previous reviewer suggested, based on the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Please, don't remove the sections in the template; even though, you might not have content for each of the at this point in time.
- [ ] You should use both `Grid` and `Flexbox` layout for the works/projects/ section.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Roseokpe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**